scripts/graphql_api_content/render_helpers.rb

```diff
@@ -10,21 +10,24 @@ def render_html(markdown)
     markdown ? CommonMarker.render_html(markdown) : nil
   end
 
-  def render_of_type(type, nullable = true, size = "medium")
+  def render_of_type(type, is_list = false, is_non_nullable = false, size = "medium")
```

Comment:
Fixed by correctly discerning type attributes, i.e. non-nullable & list/array, by correctly reading the nested JSON version of the schema. For example:

```json
{
              "name": "agentQueryRules",
              "description": null,
              "type": {
                "kind": "LIST",
                "name": null,
                "ofType": {
                  "kind": "NON_NULL",
                  "name": null,
                  "ofType": {
                    "kind": "SCALAR",
                    "name": "String",
                    "ofType": null
                  }
                }
              },
              "defaultValue": null,
              "isDeprecated": false,
              "deprecationReason": null
            },
```

The `render_of_type` method should render `agentQueryRules` as `[String]!` because it's nested within "LIST" and "NON_NULL" in the as specified in the JSON.